_posts/2018-10-12-strimzi-0.8.0-release.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title:  "Strimzi 0.8.0 released!"
+date: 2018-10-12
+author: paolo_patierno
+---
+
+We are delighted to announce the new Strimzi 0.8.0 release with many awesome new features!
+
+<!--more-->
+
+# Access your Kafka cluster from outside—without TLS
+
+The previous release added the ability to access your Kafka cluster from the outside.
+So, Kafka clients could exchange messages even if they were not running on the same Kubernetes or OpenShift cluster as the Kafka deployment.
+Previously, the TLS protocol needed to be enabled on the client side because all the provided `listeners` could only be accessed through encryption.
+With this new release, all the `listeners` have TLS enabled by default but you can choose to disable it by setting the `tls` property to `false`.
+Of course, disabling TLS is possible only for `loadbalancer` and `nodeport` listeners but not on OpenShift `route` for which the TLS is mandatory.
```

Comment:
People who don't know OpenShift in detail will not know why of course. I think you should either leave it just as "Disabling TLS is possibly only on loadbalancer and nodeport listerners" or explain that for OpenShift routes the Traffic is sneaked through as HTTPS with TLS passthrough and that is why it has to always use TLS.